browser-test/src/support/index.ts

```diff
@@ -231,7 +231,17 @@ export const gotoEndpoint = async (page: Page, endpoint: string) => {
 
 export const logout = async (page: Page) => {
   await page.click('text=Logout')
-  await page.click('button[name="logout"]') // OIDC confirmation page.
+  try {
+    const pageContent = await page.textContent('html', {timeout: 100})
+    if (pageContent!.includes('Do you want to sign-out from')) {
+      // OIDC central provider confirmation page
+      await page.click('button:has-text("Yes")')
+    }
+  }
+  catch (e) {
```

Comment:
Is error here expected/normal?

Reply:
No, but the innertext selector could theoretically return null/reject the promise. (aka it never loads).  
We do the same check in https://github.com/civiform/civiform/blob/5d7c2d5e09b699258994606620023ed9c235208c/browser-test/src/support/admin_api_keys.ts

It doesn't really matter, though.  The follow-up check "page.waitForURL" would catch the failure to logout